cloud.foundry.cli/src/main/java/cloud/foundry/cli/crosscutting/logging/Log.java

```diff
@@ -0,0 +1,125 @@
+package cloud.foundry.cli.crosscutting.logging;
+
+import java.util.logging.Level;
+
+/**
+ * The central logging module, providing a standardized interface for all logging activities in the project.
+ * By using this module instead of using custom logging facilities (like System.{out,err}.print*, for instance), a
+ * consistent style is ensured and log messages can be filtered properly.
+ * The interface is similar to Python's logging module, which provides a simple-to-use yet powerful API.
+ */
+public class Log {
+    // we can internally use the Java logging facilities
+    private static final java.util.logging.Logger logger;
+
+    // initialize and configure logger initially
```

Comment:
I would remove this comment 
-> Captain obvious (:

Reply:
True. I know many people new to `static { ... }`, though. Should I remove that comment again?

Reply:
Done.